src/main.js

```diff
@@ -160,6 +150,7 @@ let load = loader({
           signatureValidator,
           sentryManager,
           statsum:            cfg.app.statsum,
+          monitor:            monitor.prefix('api-context'),
```

Comment:
yes, it is repeated in tc-queue as well, I guess, it is generally added to the context. 
Can you tell me what this api-context does in general?